lib/main_common.pm

```diff
@@ -694,7 +694,7 @@ sub load_nfv_master_tests {
 
 sub load_nfv_trafficgen_tests {
     boot_hdd_image();
-    loadtest "nfv/moongen_installation";
```

Comment:
it was only place where moongen_installation called. not push you to delete it but just making sure that you aware that currently it is orphaned

Reply:
I know. I want to keep the option to use moongen, but default will be Trex.